portfolio/src/main/webapp/css/custom.css

```diff
@@ -0,0 +1,97 @@
+@charset "UTF-8";
+/*!
+* Start Bootstrap - Agency v6.0.2 (https://startbootstrap.com/template-overviews/agency)
+* Copyright 2013-2020 Start Bootstrap
+* Licensed under MIT (https://github.com/StartBootstrap/startbootstrap-agency/blob/master/LICENSE)
+*/
+/*!
+ * Bootstrap v4.5.0 (https://getbootstrap.com/)
+ * Copyright 2011-2020 The Bootstrap Authors
+ * Copyright 2011-2020 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+/* button  - #006600
+Button hover -  #004d00
+Button outline - #003300
+Text color -  #006600
+Gray - #6c757d
+*/
```

Comment:
Is this necessary to keep since it's commented out?

Reply:
deleted